File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.js
class Solution {
    /**
     * @param {number[]} numbers
     * @param {number} target
     * @return {number[]}
     */
    twoSum(numbers, target) {
        const n = numbers.length
        let l=0, r = n-1;
        while (l<r){
            const total = numbers[l] +numbers[r]
            if(total > target)
                r--;
            if(total < target)
                l++;
            if(total == target)
                return [l+1, r+1];
        }
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int result;
        for(int i=0;i<nums.Length;i++){
            for(int j=i+1;j<nums.Length;j++){
                result = nums[i]+nums[j];
                if(result == target){
                    return new int[] {i,j};
                }
            }
        }
        return new int[0];
    }
}
